app/assets/javascripts/modules/all-content-finder.js

```diff
@@ -10,10 +10,12 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       this.$module = $module
       this.$form = $module.querySelector('.js-all-content-finder-form')
       this.$taxonomySelect = $module.querySelector('.js-all-content-finder-taxonomy-select')
+      this.default_sort_orders = ['relevance', 'most-viewed']
```

Comment:
I was going to use a class field for this but Standard isn't a fan!

Reply:
I think you should capitalise this to represent it as a constant (or if that's not appealing change it to camel case - as the snake case looks inconsistent)

Reply:
Gah - I had it capitalised when I tried to use it as a class field, but then regressed into Ruby naming when that didn't work! 🤦